test/jake/jakelib/migrate-test.js

```diff
@@ -717,6 +717,40 @@ test('migrate runs "esm" modules', (temp) => {
   });
 });
 
+test('migrate:unlock restores lock table to one row with is_locked=0', (temp) => {
+  const db = new sqlite3.Database(`${temp}/db`);
+  // Seed multiple rows into the lock table
+  return new Promise((resolve, reject) =>
```

Comment:
Could you please use async/await instead of a promise chain?

Reply:
yep, can do that, was just trying to maintain consistency with other tests in this file

Reply:
this is legacy set of tests, I really need to add comment pointing into non-obsolete tests as a reference.